.github/workflows/ci.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        kubernetes-version: [ "v1.17.17", "v1.22.0" ]
+        kubernetes-version: [ "v1.17.17", "v1.22.0", "v1.25.3" ]
```

Comment:
v1.17.17 over ubuntu-latest will fail 😢 you need to pin 20.04

Reply:
Just realized it... 😅